sdlf-cicd/nested-stacks/template-cicd-cfn-module.yaml

```diff
@@ -139,8 +139,13 @@ Resources:
       EncryptionKey: !Ref pKMSKey
       VpcConfig: !If
         - RunInVpc
-        - SecurityGroupIds: !Split [",", !ImportValue sdlf-cicd-prerequisites-vpc-security-groups]
-          Subnets: !Split [",", !ImportValue sdlf-cicd-prerequisites-vpc-subnets]
```

Comment:
weird formatting change unrelated to GitLab?

Reply:
not sure but for now I leave as it is